services/alerts/src/AlertsManagerContext.ts

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Alert, AlertsManager, AlertsManagerAlert } from './types'
+
+type SetAlertsFunction = React.Dispatch<React.SetStateAction<Alert[]>>
+
+export const makeAlertManager = (
+    setAlerts: SetAlertsFunction
+): AlertsManager => {
+    let id = 0
```

Comment:
Keeping track of an `id` now which is handy when mapping over alerts.

Reply:
👍 :tada:  also good for finding alerts by id when removing (see other comment)